week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,64 @@
+
+const blueBtn = document.getElementById('blueBtn')
+const orangeBtn = document.getElementById('orangeBtn')
+const greenBtn = document.getElementById('greenBtn')
+
+const jumbotron = document.querySelector('.jumbotron')
+const donateBtn = document.querySelector('.buttons').children[0]
+const volunteerBtn = document.querySelector('.buttons').children[1]
+
+
+blueBtn.addEventListener('click', () => {
+    jumbotron.style.backgroundColor = "#588fbd"
+    donateBtn.style.backgroundColor = "#ffa500"
+    volunteerBtn.style.backgroundColor = "black"
+    volunteerBtn.style.color = "white"
+})
+
+orangeBtn.addEventListener('click', () => {
+    jumbotron.style.backgroundColor = "#f0ad4e"
+    donateBtn.style.backgroundColor = "#5751fd"
+    volunteerBtn.style.backgroundColor = "#31b0d5"
+    volunteerBtn.style.color = "white"
+})
+
+greenBtn.addEventListener('click', () => {
+    jumbotron.style.backgroundColor = "#87ca8a"
+    donateBtn.style.backgroundColor = "black"
+    volunteerBtn.style.backgroundColor = "#8c9c08"
+})
```

Comment:
looks good
there is a room for improvement 
all event listner do same thing here
maybe change color to array of object (or array only)?